fairlearn/postprocessing/_constants.py

```diff
@@ -11,6 +13,45 @@
 
 DEMOGRAPHIC_PARITY = "demographic_parity"
 EQUALIZED_ODDS = "equalized_odds"
+ACCURACY_SCORE = "accuracy_score"
+
+def confusion_matrix_summary(*, true_positives, false_positives, true_negatives, false_negatives):
```

Comment:
Any chance we could reuse something that already exists for this? Perhaps https://scikit-learn.org/stable/modules/generated/sklearn.metrics.confusion_matrix.html is of use?

Reply:
This is doing something very different. It is taking an already calculated confusion matrix and filling in the names of implied fields. Would it help to rename this utility function?

Reply:
I kind of understood why it's there better after reading the entire PR. You're mapping the various objectives to these metrics inside postprocessing, so this is required. I think it's fine the way it is, but it would be awesome to have a short description saying why this is needed (i.e. the second sentence of this very comment, but perhaps more nicely written?)

Reply:
Adding a more detailed docstring.